source/backend/entities/Partials/User.cs

```diff
@@ -18,6 +18,12 @@ public partial class PimsUser : StandardIdentityBaseAppEntity<long>, IDisableBas
         [NotMapped]
         public override long Internal_Id { get => UserId; set => UserId = value; }
 
+        /// <summary>
+        /// get - User is Contractor
+        /// </summary>
+        [NotMapped]
+        public bool IsContractor => UserTypeCode is not null && UserTypeCode == EnumUserTypeCodes.CONTRACT.ToString();
```

Comment:
unlike the above, I don't think we have an issue with data loading here since we should always have the userTypeCode.

Slight preference for implementing as an extension method since this field is not in the database and may confuse someone.

Reply:
ok